devito/at_controller.py

```diff
@@ -0,0 +1,303 @@
+import os
+import re
+import stat
+import math
+import shutil
+import subprocess
+import multiprocessing
+
+# global vars.
+# report folder will be created in current dir. Change if necessary
+reports_folder_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "auto-tune-reports")
+
+# where best block sizes for each kernel will be kept
+final_report_path = os.path.join(reports_folder_path, "final_report.txt")
+
+
+# Gets the best block sizes for given kernel if they are in the final_report_path
+# param - int time order - time order of kernel.
+# param - int space_order - space order of kernel.
+# returns - list - best block size. Starting from outer most dimension
+# returns - None - if report does not exist or does not contain required block sizes
```

Comment:
This, as well as all other function documentation comments in this PR, should really be a standard Python docstring.